manual/doc/installation.tex

```diff
@@ -31,7 +31,7 @@ \section{Installing the Dependencies}
 If you intend to use the Python \cite{Python} interface to the EOS library, you will need
 to additionally install
 \begin{description}
-    \item[python3-dev] python3 interpreter and required header files,
+    \item[python3-dev] python3 interpreter in version 3.3 or higher, and required header files,
```

Comment:
I would not support 3.3. That's very old and many systems like conda forge don't support it either (I think for numpy reasons). Let's go with 3.4 or 3.5, the most recent is 3.6. In any case python 3 may be a stumbling block for mac users where the stock python is 2.7. These users would find it easiest to use conda with python 3, so we need to make sure eos plays well with conda

Reply:
Just checked, 3.5 should be enough

Reply:
Ok:

- CentOS 7 still has only Python 2.7 in the base repository. Requires a new installation anyway. And I still see problems with libboost_python in this case.
- Python 3.{4,5,6} is available pre-built for MacOS X.

So I see no problem to go with >=3.5 here.